packages/web-components/fast-foundation/src/design-token/design-token.spec.ts

```diff
@@ -1,12 +1,12 @@
 
-import { css, DOM, FASTElement, Observable } from "@microsoft/fast-element";
+import { css, DOM, FASTElement, html, Observable } from "@microsoft/fast-element";
 import { expect } from "chai";
 import { DesignSystem } from "../design-system";
 import { FoundationElement } from "../foundation-element";
 import { DesignToken } from "./design-token";
 
 new DesignSystem().register(
-    FoundationElement.compose({ type: class extends FoundationElement { }, baseName: "custom-element" })()
+    FoundationElement.compose({ type: class extends FoundationElement { }, template: html`<slot></slot>`, baseName: "custom-element" })()
```

Comment:
I think you'll need to update this code after you update the branch since the DesignSystem API changed a bit. See the DesignSystem tests for some examples. Shouldn't be difficult to update.